### DISCUSSION TITLE
Issues when using python script to test the trained model

### DISCUSSION THREAD

**Discussion by sd123s0-3123** [deepmodeling/deepmd-kit#2852] [Q&A] [answered]
I was following the instractionn of the website: https://nb.bohrium.dp.tech/detail/1003 on my own ubuntu system(22.03), some issue occur when I reach part 5 of testing trained model. when I try to run the python script to generate a plot to compare trained data and validation data, the following error occurred:
WARNING:tensorflow:From /home/murphy/deepmd-kit/lib/python3.10/site-packages/tensorflow/python/compat/v2_compat.py:107: disable_resource_variables (from tensorflow.python.ops.variable_scope) is deprecated and will be removed in a future version.
Instructions for updating:
non-resource variables are not supported in the long term
WARNING:root:To get the best performance, it is recommended to adjust the number of threads by setting the environment variables OMP_NUM_THREADS, TF_INTRA_OP_PARALLELISM_THREADS, and TF_INTER_OP_PARALLELISM_THREADS. See https://deepmd.rtfd.io/parallelism/ for more information.
Traceback (most recent call last):
  File "/home/murphy/CH4/01.train/correlation.py", line 6, in <module>
    predict = training_systems.predict("graph.pb")
  File "/home/murphy/deepmd-kit/lib/python3.10/site-packages/dpdata/system.py", line 947, in predict
    driver = Driver.get_driver(driver)(*args, **kwargs)
  File "/home/murphy/deepmd-kit/lib/python3.10/site-packages/dpdata/plugins/deepmd.py", line 369, in __init__
    import deepmd.DeepPot as DeepPot
  File "/home/murphy/deepmd-kit/lib/python3.10/site-packages/deepmd/__init__.py", line 32, in <module>
    set_mkl()
  File "/home/murphy/deepmd-kit/lib/python3.10/site-packages/deepmd/env.py", line 244, in set_mkl
    if "mkl_rt" in np.__config__.get_info("blas_mkl_info").get("libraries", []):
AttributeError: module 'numpy.__config__' has no attribute 'get_info'

Will it be the module predict goes wrong?

**Comment by njzjz** [comment]
#2853 is going to fix it.

**Reply by njzjz** [reply, marked as answer]
By the way, TensorFlow has limited the maximum NumPy version to 1.24.3.
https://github.com/tensorflow/tensorflow/blob/v2.13.0/tensorflow/tools/pip_package/setup.py#L93
If you installed TensorFlow normally, NumPy 1.26 should not be installed.

**Reply by sd123s0-3123** [reply]
Thanks for your reply，I successfully ran the script after reinstalling the 1.24.3 version of numpy and got the diagram.
